server/proxy/config.ini

```diff
@@ -15,5 +15,8 @@ BitmapUpdate = 1
 NlaSupport = 0
 TlsSupport = 0
 RdpSupport = 0
+
+[Channels]
+Mode = 1 # 0 = Whitelist, 1 = Blacklist
```

Comment:
I'd call this WhitelistMode (indicating that the default if unspecified is blacklist)